src/StatusQ/Core/Theme/ThemePalette.qml

```diff
@@ -155,8 +155,12 @@ QtObject {
         property color secondaryBackgroundColor
     }
 
+    // TODO: put colors to palette
     property QtObject statusSwitchTab: QtObject {
-        property color backgroundColor
+        property color buttonBackgroundColor: "#4360DF"
+        property color barBackgroundColor: "#1A4360DF"
+        property color selectedTextColor: white
+        property color textColor: "#4360DF"
```

Comment:
Are we sure these are exactly the same in both themes?
Also, can we move these colors to `StatusColorPalettes`?

Reply:
@MishkaRogachev https://github.com/status-im/status-desktop/issues/5612#issuecomment-1119280046

Reply:
Updated on palette colors. Btw in my opinion it will be better to override active tab color for dark theme on primary color from light theme